docs/conf.py

```diff
@@ -0,0 +1,69 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=invalid-name
+
+"""
+Sphinx documentation builder
+"""
+
+# General options:
+import os
+
+project = "Template project"
+copyright = "2022"  # pylint: disable=redefined-builtin
+author = ""
+
+# The short X.Y version
+version = ""
+# The full version, including alpha/beta/rc tags
+with open(
+    f"{os.path.dirname(os.path.abspath(__file__))}/"
+    f"../prototype_template/VERSION.txt",
```

Comment:
It might be nice to use `os.path.join` here.

```suggestion
    os.path.join(
        os.path.dirname(os.path.abspath(__file__)),
        "..",
        "prototype_template",
        "VERSION.txt",
    ),
```

This should, in theory, be more portable since it no longer assumes the path separator is `/`.

Reply:
sweet!